scripts/treedb_vectordbbench_artifact_test.py

```
@@ -4,17 +4,165 @@
 
 
+@functools.cache
+def go_root() -> Path:
+    return Path(subprocess.run(
+        ("go", "env", "GOROOT"),
+        stdout=subprocess.PIPE,
+        stderr=subprocess.PIPE,
+        text=True,
+        timeout=10,
+        check=True,
+    ).stdout.strip())
```

Ruff (0.16.2)

[warning] Prefer `capture_output` over sending `stdout` and `stderr` to `PIPE`

Replace with `capture_output` keyword argument

(UP022)